core/store/src/flat/metrics.rs

```diff
@@ -73,21 +73,24 @@ impl FlatStorageReshardingShardSplitMetrics {
         left_child_shard: ShardUId,
         right_child_shard: ShardUId,
     ) -> Self {
-        use flat_state_metrics::*;
+        use resharding::flat_state_metrics;
+
         let parent_shard_label = parent_shard.to_string();
         let left_child_shard_label = left_child_shard.to_string();
         let right_child_shard_label = right_child_shard.to_string();
         Self {
             parent_shard,
             left_child_shard,
             right_child_shard,
-            parent_status: resharding::STATUS.with_label_values(&[&parent_shard_label]),
-            left_child_status: resharding::STATUS.with_label_values(&[&left_child_shard_label]),
-            right_child_status: resharding::STATUS.with_label_values(&[&right_child_shard_label]),
-            split_shard_processed_batches: resharding::SPLIT_SHARD_PROCESSED_BATCHES
+            parent_status: flat_state_metrics::STATUS.with_label_values(&[&parent_shard_label]),
```

Comment:
nit: Do we need changes in this file?

Reply:
it is to re-order the metrics module as @wacban suggested.